crates/test-collector/src/lib.rs

```diff
@@ -61,8 +61,12 @@ pub enum ExpectedTestResult {
     Panics(ExpectedPanicValue),
 }
 
+pub trait ForkConfig {}
+
+impl ForkConfig for RawForkConfig {}
+
 #[derive(Debug, Clone, PartialEq)]
-pub enum ForkConfig {
+pub enum RawForkConfig {
     Id(String),
     Params(String, BlockId),
```

Comment:
```suggestion
    Params(RawForkParams),
```
please create a separate type and use it for ForkTargets as well. In other comment i suggested making it a pipeline, first function should map from `RawForkConfig` to `RawForkParams` (querying scarb config). And then other function should parse url in params.

Reply:
Okay, that convinces me to provide worse UX xD